deepchecks/checks/performance/simple_model_comparison.py

```diff
@@ -126,27 +131,30 @@ def _find_score(self, train_ds: Dataset, test_ds: Dataset, task_type: ModelType,
             elif task_type in (ModelType.BINARY, ModelType.MULTICLASS):
                 clf = DecisionTreeClassifier(max_depth=self.max_depth,
                                              random_state=self.random_state, class_weight='balanced')
-            if clf:
+
+            if clf: # FIXME: is 'clf' possibly unbound?
```

Comment:
looks like instead of this if it need to be an `else: raise exception` after the elif

Reply:
updated